.github/workflows/benchmark.yaml

```diff
@@ -25,6 +25,6 @@ jobs:
         run: npm run build
 
       - name: Benchmark
-        run: npm run benchmark --concurrency 1
+        run: npm run benchmark
```

Comment:
```suggestion
        run: npm run benchmark --concurrency=1
```
will solve the error. We need them to run one after the other to get more stable performance metrics